plugins/development/plugin-git/src/node/utils/checkGitRepo.ts

```diff
@@ -11,3 +12,26 @@ export const checkGitRepo = (cwd: string): boolean => {
     return false
   }
 }
+
+export const checkGitRepoType = (cwd: string): GitType | null => {
```

Comment:
You are creating a git repo with `git init`, so repo type is a bad name to me.

Also I am wondering if we should check the first remote if origin doesn't exist?

Reply:
If do it this way, we will need to execute multiple command lines here: first, use `git remote` to list all of them, then get the first one, and finally execute `git remote get-url xxx`.